addons/point_of_sale/static/src/app/models/pos_order_line.js

```diff
@@ -486,8 +482,12 @@ export class PosOrderline extends Base {
     }
 
     get_all_prices(qty = this.get_quantity()) {
-        const product = this.get_product();
         const priceUnit = this.get_unit_price();
+        return this._get_all_prices({ qty, priceUnit });
+    }
+
+    _get_all_prices({ qty, priceUnit, special_mode = false }) {
```

Comment:
extracted to allow calculation of `tax-included` prices even if the taxes are price-excluded.